lib/Default/MySqlDatabase.class.php

```diff
@@ -1,37 +1,28 @@
 <?php declare(strict_types=1);
-require_once(CONFIG . 'SmrMySqlSecrets.inc');
 
 abstract class MySqlDatabase {
-	// add configuration static members via traits
-	use SmrMySqlSecrets;
-
 	protected static $dbConn;
 	protected static $selectedDbName;
 	protected $dbResult = null;
 	protected $dbRecord = null;
 
-	public function __construct($dbName) {
+	public function __construct($dbName=null, MySqlProperties $mysqlProperties=null) {
```

Comment:
```suggestion
	public function __construct($dbName = null, MySqlProperties $mysqlProperties = null) {
```
Spacing for Scrutinizer.